### DISCUSSION TITLE
Do all team members need lfs to retrieve resources tracked by lfs?

### DISCUSSION THREAD

**Discussion by Lauro235** [git-lfs/git-lfs#5472] [Q&A]
I imagine that all team members need to download individual copies of LFS to ensure they can access the resources managed by git lfs, but it would be great to get confirmation on this.

It would also be cool for the documentation to reflect what lfs would look like in a team scenario.

Thanks!

**Comment by bk2204** [comment]
Yes, typically team members need to download individual copies of LFS to ensure they can access the resources necessary.  If some team members don't need to access the LFS resources, then it's fine for them to simply use `--skip-smudge` in their configuration and not download those files (say, if only images are tracked in Git LFS and a user doesn't need to interact with those).

What kind of documentation were you hoping for for a team scenario?  The goal is that Git LFS works mostly automatically and very similarly to Git without Git LFS, such that the workflow is not very different for most scenarios.
